src/app/pagination/pagination.component.ts

```diff
@@ -32,7 +32,6 @@ export class MzPaginationComponent extends HandlePropChanges implements OnInit {
 
   changeCurrentPage(pageNumber: number) {
     this.currentPage = pageNumber;
-    this.renderButtons();
```

Comment:
Since this has been removed buttons are not refreshing when current page is not in the range of rendered buttons

![pagination-playground-02](https://user-images.githubusercontent.com/6987084/37257682-42a9e608-2543-11e8-9885-484c290f56f5.gif)